### ISSUE TITLE
Looking for LLaMA 2?

### ISSUE THREAD

**Issue by carmocca** [Lightning-AI/lit-llama#452]
Meta AI has since released LLaMA 2. Additionally, new Apache 2.0 licensed weights are being released as part of the [Open LLaMA project](https://github.com/openlm-research/open_llama).

To run LLaMA 2 weights, Open LLaMA weights, or Vicuna weights (among other LLaMA-like checkpoints), **check out the [Lit-GPT repository](https://github.com/Lightning-AI/lit-gpt)**.

**Comment by rasbt** [comment]
Yes, full finetuning is supported via [finetune/full.py](https://github.com/Lightning-AI/lit-gpt/blob/main/finetune/full.py) script given a Llama 2 model provided via the `--checkpoint_dir`.

You can also use a custom dataset given that you prepare it in the right format. You can see the `prepare_*.py` scripts [here](https://github.com/Lightning-AI/lit-gpt/tree/main/scripts) for guidance.

**Comment by cabal-daniel** [comment]
Is this repo still intended to be supported given since it seems like the lit-gpt repo supports more and newer models.
